demo-shell/package.json

```diff
@@ -124,6 +124,7 @@
     "rimraf": "^2.6.2",
     "ts-node": "~4.1.0",
     "tslint": "~5.9.1",
-    "typescript": "~2.5.3"
+    "typescript": "~2.5.3",
+    "webpack": "3.10.0"
```

Comment:
do we need to add it here?

Reply:
I reverted this change. I thought that it was needed because locally I could not start the demo shell without that, but I've just checked on another computer and all good without that line - so it seems it is not needed.